server/superdesk/io/email.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import imaplib
+from .ingest_service import IngestService
+from superdesk.io import register_provider
+from superdesk.upload import url_for_media
+
+from superdesk.io.rfc822 import rfc822Parser
+
+PROVIDER = 'email'
+
+
+class EmailReaderService(IngestService):
+
+    def __init__(self):
+        self.parser = rfc822Parser()
+
+    def _update(self, provider):
+        server = provider.get('config', {}).get('server', None)
+        if not server:
+            server = ''
+        port = provider.get('config', {}).get('port', None)
+        if not port:
+            port = 993
+
```

Comment:
We can have something like this `port = provider.get('config', {}).get('port', 993)`